app/src/main/res/layout/bottom_sheet_event.xml

```diff
@@ -296,7 +297,9 @@
                     android:text="@string/sign_up"
                     android:textAllCaps="false"
                     android:textColor="@color/accent_yellow"
-                    android:textStyle="bold">
+                    android:textStyle="bold"
+                    android:visibility="gone"
+                    tool:visibility="gone">
```

Comment:
tools:visibility="visible" so developers know it's there

Reply:
Updated